cdc/generation.hh

```diff
@@ -118,6 +118,23 @@ public:
     {}
 };
 
+enum class stream_kind : int8_t {
```

Comment:
"kind" is not a good fit here, they all have the same kind, maybe "disposition",